### ISSUE TITLE
Accessibility Errors in byc4nyc

### ISSUE THREAD

**Issue by cycomachead** [bjc-edc/bjc-r#272]
Cut and Paste from recent CI run:

```
Failures:

  1) bjc4nyc - nyc_bjc/1-intro-loops (/topic/topic.html) : is WCAG 2.0 accessible
     Failure/Error:
       expect(page).to be_axe_clean
         .according_to(*required_a11y_standards)
         .skipping(*skipped_rules)
         .excluding(*excluded_elements)

       Found 1 accessibility violation:

       1) list: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (serious)
           https://dequeuniversity.com/rules/axe/4.10/list?application=axeAPI
           The following 2 nodes violate this rule:
           
               Selector: section:nth-child(3) > ol
               HTML: <ol>
               Fix all of the following:
               - List element has direct children that are not allowed: img
               
               Selector: section:nth-child(5) > ol
               HTML: <ol>
               Fix all of the following:
               - List element has direct children that are not allowed: img
               
       Invocation: axe.run({:exclude=>[["[data-a11y-errors=\"true\"]"], [".todo"], [".comment"], [".commentBig"], [".ap-standard"], [".csta-standard"], ["var"]]}, {:runOnly=>{:type=>:tag, :values=>[:wcag2a, :wcag2aa]}}, callback);
     # ./utilities/specs/accessibility_spec.rb:98:in `block (2 levels) in a11y_test_cases'

  2) bjc4nyc - nyc_bjc/2-conditionals-abstraction (/topic/topic.html) : is WCAG 2.0 accessible
     Failure/Error:
       expect(page).to be_axe_clean
         .according_to(*required_a11y_standards)
         .skipping(*skipped_rules)
         .excluding(*excluded_elements)

       Found 1 accessibility violation:

       1) list: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (serious)
           https://dequeuniversity.com/rules/axe/4.10/list?application=axeAPI
           The following 2 nodes violate this rule:
           
               Selector: section:nth-child(1) > ol
               HTML: <ol>
               Fix all of the following:
               - List element has direct children that are not allowed: img
               
               Selector: section:nth-child(5) > ol
               HTML: <ol>
               Fix all of the following:
               - List element has direct children that are not allowed: img
               
       Invocation: axe.run({:exclude=>[["[data-a11y-errors=\"true\"]"], [".todo"], [".comment"], [".commentBig"], [".ap-standard"], [".csta-standard"], ["var"]]}, {:runOnly=>{:type=>:tag, :values=>[:wcag2a, :wcag2aa]}}, callback);
     # ./utilities/specs/accessibility_spec.rb:98:in `block (2 levels) in a11y_test_cases'

  3) bjc4nyc - nyc_bjc/1-intro-loops (/cur/programming/1-introduction/3-drawing/4-modify-your-pinwheel.html) : is WCAG 2.0 accessible
     Failure/Error:
       expect(page).to be_axe_clean
         .according_to(*required_a11y_standards)
         .skipping(*skipped_rules)
         .excluding(*excluded_elements)

       Found 1 accessibility violation:

       1) list: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (serious)
           https://dequeuniversity.com/rules/axe/4.10/list?application=axeAPI
           The following 1 node violate this rule:
           
               Selector: ol[start="5"]
               HTML: <ol start="5">
               Fix all of the following:
               - List element has direct children that are not allowed: div
               
       Invocation: axe.run({:exclude=>[["[data-a11y-errors=\"true\"]"], [".todo"], [".comment"], [".commentBig"], [".ap-standard"], [".csta-standard"], ["var"]]}, {:runOnly=>{:type=>:tag, :values=>[:wcag2a, :wcag2aa]}}, callback);
     # ./utilities/specs/accessibility_spec.rb:98:in `block (2 levels) in a11y_test_cases'

Finished in 2 minutes 49.7 seconds (files took 0.68836 seconds to load)
183 examples, 55 failures
```

Unit 1 only

**Comment by cycomachead** [comment]
This is just to test the GitHub Copilot AI fixing thing

**Comment by cycomachead** [comment]
GenAI so far failed at auto fixing accessibility issues. Shame.
